rails-snow-queen/config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  resources :quotes, only: [:new] 
+  resources :quotes, only: [:new, :create] 
+  get('/', { to: 'quotes#new'})
```

Comment:
https://guides.rubyonrails.org/routing.html#using-root
```suggestion
  root 'quotes#new'
```

Reply:
done